src/main/java/com/appdirect/sdk/vendorFields/fieldsValidation/VendorFieldValidationController.java

```diff
@@ -0,0 +1,47 @@
+package com.appdirect.sdk.vendorFields.fieldsValidation;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+import static org.springframework.web.bind.annotation.RequestMethod.POST;
+
+import java.util.Map;
+import java.util.concurrent.Callable;
+
+import lombok.RequiredArgsConstructor;
+
+import org.springframework.web.bind.WebDataBinder;
+import org.springframework.web.bind.annotation.InitBinder;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.appdirect.sdk.vendorFields.converter.FlowTypeConverter;
+import com.appdirect.sdk.vendorFields.converter.OperationTypeConverter;
+import com.appdirect.sdk.vendorFields.fieldsValidation.model.VendorFieldsValidationResponse;
+import com.appdirect.sdk.vendorFields.model.FlowType;
+import com.appdirect.sdk.vendorFields.model.OperationType;
+
+/**
+ * Defines the endpoint for enforcing vendor required requiredFields on their products
+ */
+@RestController
+@RequiredArgsConstructor
+public class VendorFieldValidationController {
+	private final VendorFieldValidationHandler vendorFieldValidationHandler;
+
+	@RequestMapping(method = POST,
+			value = "/api/v1/admin/internal/vendorvalidations/",
```

Comment:
This is not internal:
```suggestion
			value = "/api/v1/admin/vendorvalidations/",
```

Reply:
Done